lib/start.js

```diff
@@ -2,20 +2,22 @@ const openBrowser = require('react-dev-utils/openBrowser')
 const webpack = require('webpack')
 const WebpackDevServer = require('webpack-dev-server')
 const chalk = require('chalk')
+const initializeMiddleware = require('./initializeMiddleware')
+const express = require('express')
 
 const webpackConfig = require('./webpack.config')
 
 module.exports = ({ args, getPackage }) => {
+  const app = express()
   const pkg = getPackage()
-  const publicPath = args.contextPath || pkg['context-path']
+  const publicPath = pkg['context-path']
 
-  const main = args.main || pkg.main
   const env = process.env.NODE_ENV || args.env || 'development'
   const port = process.env.PORT || args.port || 8080
   const host = process.env.HOST || args.host || 'localhost'
   const proxy = process.env.PROXY || args.proxy || 'https://localhost:8993'
 
-  const config = webpackConfig({
+  const { client, server } = webpackConfig({
```

Comment:
I think we should change this to return the server or the client config, not both.